app/src/screens/Splash.tsx

```diff
@@ -10,8 +10,7 @@ import {
 import { useAgent } from '@aries-framework/react-hooks'
 import { agentDependencies } from '@aries-framework/react-native'
 import AsyncStorage from '@react-native-async-storage/async-storage'
-import { useNavigation } from '@react-navigation/core'
-import { CommonActions } from '@react-navigation/native'
+import { CommonActions, useNavigation } from '@react-navigation/native'
```

Comment:
Curious, why are we switching from core to native here?

Reply:
Just saw your other PR in Bifold, you can ignore the above comment. Does this PR depend on that PR at all then?

Reply:
No dependency and that one will be more complicated. What happened was we were mixing and matching where `useNavigation` came from. Some times `core` and others `native`. But the hooks need (in BC Wallet's case) to be one or the other - not both. I settled on the one I though was best. Opinions welcome.